src/Shell/Task/ExtractTask.php

```diff
@@ -321,6 +332,9 @@ public function getOptionParser()
         ])->addOption('merge', [
             'help' => 'Merge all domain strings into the default.po file.',
             'choices' => ['yes', 'no']
+        ])->addOption('relative-paths', [
+            'help' => 'Use relative paths in the .pot file',
+            'choices' => ['yes', 'no']
```

Comment:
This could be a `boolean` argument. That will default it to `false` and when `--relative-paths` is set on the CLI it will be `true`.